devices/ubisys.js

```diff
@@ -536,15 +536,19 @@ module.exports = [
         model: 'S1',
         vendor: 'Ubisys',
         description: 'Power switch S1',
-        exposes: [e.switch(), e.power().withAccess(ea.STATE_GET).withEndpoint('meter').withProperty('power'),
+        exposes: [
+            e.switch(),
             e.action([
                 'toggle', 'on', 'off', 'recall_*',
                 'brightness_move_up', 'brightness_move_down', 'brightness_stop',
             ]),
-            e.power_on_behavior(), e.energy()],
+            e.power_on_behavior(),
+            e.power().withAccess(ea.STATE_GET).withEndpoint('meter').withProperty('power'),
```

Comment:
You probably don't the `meter` endpoint postfix. This can be achieved now with https://github.com/Koenkk/zigbee-herdsman-converters/pull/5133.

e.g.
```
meta: {multiEndpointSkip: ['power', 'energy']}
```

Reply:
If i drop the withEndpoint the fronend still displays it fine (just not indented) but the get request still end up failing the same way as before, am i missing something obvious?

energy/power is already in the state without a suffix (it always has been) so  skip doesn’t seem to be doing anything new for them.

Reply:
OK this makes sense, as `postfixWithEndpointName` is the function that  consumes multiEndpointSkip and neither the power or energy converter uses `postfixWithEndpointName`.

So I'm guessing the current PR is fine! It's just the frontend that needs to properly use the expose information.

Reply:
`fz.metering` does indeed not use `postfixWithEndpointName`. But if I understand correctly now this is only for the reading part. Ideally just `e.power().withAccess(ea.STATE_GET)` should be used. Can you try if in `endpoint()`  the following works?

```js
return {'l1': 1, 's1': 2, 'default': 3};
```

Reply:
```[hyperon :: sjorge][~/Documents/repos/zigbee-herdsman-converters][patch-8 ✗]
[■]$ git diff
diff --git a/devices/ubisys.js b/devices/ubisys.js
index 1c60a6cf..23dca0a8 100644
--- a/devices/ubisys.js
+++ b/devices/ubisys.js
@@ -543,14 +543,14 @@ module.exports = [
                 'brightness_move_up', 'brightness_move_down', 'brightness_stop',
             ]),
             e.power_on_behavior(),
-            e.power().withAccess(ea.STATE_GET).withEndpoint('meter').withProperty('power'),
-            e.energy().withAccess(ea.STATE_GET).withEndpoint('meter').withProperty('energy'),
+            e.power().withAccess(ea.STATE_GET).withProperty('power'),
+            e.energy().withAccess(ea.STATE_GET).withProperty('energy'),
         ],
         fromZigbee: [fz.on_off, fz.metering, fz.command_toggle, fz.command_on, fz.command_off, fz.command_recall, fz.command_move,
             fz.command_stop, fz.power_on_behavior, ubisys.fz.configure_device_setup],
         toZigbee: [tz.on_off, tz.metering_power, tz.currentsummdelivered, ubisys.tz.configure_device_setup, tz.power_on_behavior],
         endpoint: (device) => {
-            return {'l1': 1, 's1': 2, 'meter': 3};
+            return {'l1': 1, 's1': 2, default: 3};
         },
         configure: async (device, coordinatorEndpoint, logger) => {
             const endpoint = device.getEndpoint(3);
@@ -636,14 +636,14 @@ module.exports = [
                 'brightness_stop_s2']),
             e.power_on_behavior().withEndpoint('l1'),
             e.power_on_behavior().withEndpoint('l2'),
-            e.power().withAccess(ea.STATE_GET).withEndpoint('meter').withProperty('power'),
-            e.energy().withAccess(ea.STATE_GET).withEndpoint('meter').withProperty('energy'),
+            e.power().withAccess(ea.STATE_GET).withProperty('power'),
+            e.energy().withAccess(ea.STATE_GET).withProperty('energy'),
         ],
         fromZigbee: [fz.on_off, fz.metering, fz.command_toggle, fz.command_on, fz.command_off, fz.command_recall, fz.command_move,
             fz.command_stop, fz.power_on_behavior, ubisys.fz.configure_device_setup],
         toZigbee: [tz.on_off, tz.metering_power, ubisys.tz.configure_device_setup, tz.power_on_behavior, tz.currentsummdelivered],
         endpoint: (device) => {
-            return {'l1': 1, 'l2': 2, 's1': 3, 's2': 4, 'meter': 5};
+            return {'l1': 1, 'l2': 2, 's1': 3, 's2': 4, default: 5};
         },
         meta: {multiEndpoint: true},
         configure: async (device, coordinatorEndpoint, logger) => {
```

This causes all sorts of breakage now, I can now read power/energy but for the S1 state and power_on_behavior is broken.

With as the PR is now, I can manualy everythign works fine, except the frontend's read calls as it just ignores the withEndpoint(), I'm pretty sure this is a bug frontend side.

Reply:
Since this a a special case, an alternative solution would be to create a separate `tzLocal`/`currentsummdelivered` for these devices which always read from the correct endpoint? (`meter` endpoint)

Reply:
The expose data contains everything to do the read properly as it contains the correct endpoint, It’s just the frontend not properly parsing it.

Reply:
I see, but we should wonder wether an endpoint property without the endpoint name in it should be allowed. (that is also the reason why the tests are adapted in this pr). Ubisys devices are the only supported devices which use such construction: (like `.withEndpoint('meter').withProperty('power')`)

Reply:
I’ll see if i can update both convertGet to be `multiEndpointSkip` aware somehow. Let me thing about it a bit more.